.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
 install:
   - pip install --upgrade pip
   - pip install --upgrade numpy scipy matplotlib pandas
+  - pip install --upgrade dask
```

Comment:
Is this required?

Reply:
according https://github.com/Netflix/vmaf/pull/295 's result, it does not need.
it seems that current pip package repository has the right version, dask(1.1.0) and scikit-image(0.14.2). 
But I need do something to active the ci build, so ...

Reply:
Ok. It sounds non-essential to the vmaf package. Would you mind removing this?

Reply:
I reset the commitid to e1c4458(fix bug: crash while predictionStructs empty), the last commit that I want, and push --force,  but it seems that CI can not be re-activate to re-check it.
Do I have to commit some unwanted?

Reply:
Could you forward to the latest commit, and add a new commit just to remove that line?

Reply:
format code  to same checkstyle and activate the CI build,